components/ContextToRuntime.coffee

```diff
@@ -104,21 +112,17 @@ exports.getComponent = ->
           # Different runtime, different graph. Disconnect old runtime connection
           currentContext.runtime.disconnect()
 
+      send = _.debounce sendContext, 300, true
```

Comment:
So we don't send more than once in this situation where we happen to get the runtime just between when it had connected, but before it had received capabilities